src/kit/Form.tsx

```diff
@@ -97,12 +102,23 @@ type Form = JSX.Element & {
 };
 
 const Form = ({ noValidate = true, ...props }: FormProps): JSX.Element => {
-  return <AntdForm noValidate={noValidate} {...props} />;
+  const {
+    themeSettings: { className: themeClass },
+  } = useTheme();
+  return <AntdForm noValidate={noValidate} {...props} className={themeClass} />;
```

Comment:
please ensure that the classname in the theme is merged with the classname in the props